Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:20.04
```

Comment:
Please add a package stage to reduce image size here.

Reply:
I have made the changes